packages/it-batch/README.md

```diff
@@ -2,7 +2,8 @@
 
 [![Build status](https://travis-ci.org/achingbrain/it.svg?branch=master)](https://travis-ci.org/achingbrain/it?branch=master) [![Coverage Status](https://coveralls.io/repos/github/achingbrain/it/badge.svg?branch=master)](https://coveralls.io/github/achingbrain/it?branch=master) [![Dependencies Status](https://david-dm.org/achingbrain/it/status.svg?path=packages/it-batch)](https://david-dm.org/achingbrain/it?path=packages/it-batch)
 
-> Takes an async iterator that emits things and emits them as fixed-size batches
+> Takes an (async) iterable that emits things and returns an async iterable that
+> emits those things in fixed-sized batches.
```

Comment:
Can be on one line

Reply:
Fixed by ebabad16f043a6b3fdc0dbcf44e008510fcac282